docs/study_viewer.md

```diff
@@ -73,6 +73,7 @@ Example configuration:
             {
                 // configuration specific to the "Request Access" button
                 "type": "request_access",
+                "required_idp": "ImmPort",// optional, requires user to have specific type to request access
```

Comment:
```suggestion
                "required_idp_field": "required_idp", // optional, requires user to have logged in through this identity provider to request access
```

Reply:
The feature is a bit more complex than the current implementation: the config should be `required_idp_field`, which is set to the field in the dictionary that contains the required IDP - for NCT it's `required_idp`.

Each row of data could have `required_idp` set or not, for example one row could have it set to `immport`, another to `google` and another to nothing (so any IDP would work to request access to this one)